packages/calypso-products/src/products-list.ts

```diff
@@ -182,6 +182,21 @@ export const JETPACK_PRODUCTS_LIST_WITH_FEATURES: Record< JetpackProductSlug, Pr
 			FEATURE_SPELLING_CORRECTION_V2,
 		],
 	},
+	// Jetpack Search for WPCom
```

Comment:
These are considered Jetpack products in the rest of the codebase. To avoid confusion and fix TS issues, I'm moving them to `JETPACK_PRODUCTS_LIST_WITH_FEATURES`. Given where `JETPACK_PRODUCTS_LIST_WITH_FEATURES` is used, this seems pretty safe to do.